CONTRIBUTING.md

```diff
@@ -27,8 +27,8 @@ Contents
 ---
 
 - Related documents
-  - [Creating Operators](docs_app/content/guide/operators.md#creating-custom-operators)
-  - [Writing Marble Tests](docs_app/content/guide/testing/marble-testing.md)
+  - [Creating Operators](../../apps/rxjs.dev/content/guide/operators.md#creating-custom-operators)
```

Comment:
This isn't really working. As far as I understand, we now have `apps/rxjs.dev/` that contains (almost) all content of the old `docs_app` folder. So, I guess this should work:

```suggestion
  - [Creating Operators](apps/rxjs.dev/content/guide/operators.md#creating-custom-operators)
```